gcloud/monitoring/metric.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Metric Descriptors for the Google Monitoring API."""
+
+# Features intentionally omitted from this first version of the client library:
+#   - Creating and deleting metric descriptors.
+
+import collections
+
+from gcloud.monitoring.label import LabelDescriptor
+
+
+class MetricDescriptor(collections.namedtuple(
```

Comment:
Let's find out whether that's the right direction. All I'm pointing out is that I don't think we do this elsewhere in gcloud-python... Maybe we want to... @dhermes ?